x-pack/platform/plugins/shared/apm_sources_access/common/ui_settings.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { UiSettingsParams } from '@kbn/core-ui-settings-common';
+import { i18n } from '@kbn/i18n';
+import { OBSERVABILITY_APM_SOURCES_ACCESS_APM_SOURCES_ID } from '@kbn/management-settings-ids';
+import { indicesSchema } from './config_schema';
+
+/**
+ * uiSettings definitions for the apm_sources_access plugin.
+ */
+export const uiSettings: Record<string, UiSettingsParams> = {
```

Comment:
What do we need this for?

Reply:
To define the UI Settings that allow the `tracesContextService` to be able to get the APM Indices.

Reply:
Couldn't you just consume the `getApmIndices` function?